share/spice/just_delete_me/content.handlebars

```diff
@@ -3,13 +3,8 @@
     {{notes}}
   </p>
 {{/if}}
-{{#if delete_url}}
```

Comment:
We still need this for the detail view, otherwise we have no link to the `delete_url`.

I would change it to something like "Delete your X account: $url"

Or just make that the link text: `<a href="{{delete_ur}}">Delete my X account</a>`

Not sure which will look better